spec/views/shopping_list_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Food integration tests', type: :feature do
+  include Devise::Test::IntegrationHelpers
+
+  before(:each) do
+    @user = User.create(name: 'Nick', email: 'nick@example.com', password: '123456', password_confirmation: '123456')
+    @food = Food.create(name: 'Apple', user_id: @user.id, measurement_unit: 'grams', price: 10, quantity: 1)
+    @recipe = Recipe.create(name: 'Test', preparation_time: '1 hr', cooking_time: '1 hr', description: 'Test',
+                            public: true, user_id: @user.id)
+    @recipe_food = RecipeFood.create(quantity: 1, recipe_id: @recipe.id, food_id: @food.id)
+    CartItem.create(
+      name: @food.name,
+      measurement_unit: @food.measurement_unit,
+      price: @food.price,
+      quantity: @recipe_food.quantity,
+      recipe: @recipe,
+      user: @user
+    )
+    login_as(@user, scope: :user)
+  end
+
+  describe 'index page' do
+    before { visit shopping_list_path }
```

Comment:
- [OPTIONAL] The same case(mentioned above) is also applied here 👆 These tests were also failing in the last review. Could you please fix this instead of removing them? 💪📌

   _You can reach out to me on slack if you need any help in this and we can work on this together_